falcon/request.py

```diff
@@ -182,7 +182,12 @@ class Request(object):
 
             Note:
                 `req.path` may be set to a new value by a `process_request()`
-                middleware method in order to influence routing.
+                middleware method in order to influence routing. If the
+                original path requested was url-encoded, it will be decoded
+                before being returned this attribute. If this attribute is to
```

Comment:
> If the original path requested was url-encoded, it will be decoded before being returned :question: this attribute.

It feels like some preposition is missing here in-between, like *it will be decoded before being returned **as/in/by** this attribute*.

Reply:
I went ahead and addressed this.